docs/config.json

```diff
@@ -246,25 +246,25 @@
         }
       ],
       "url": "#",
-      "key": "docker",
```

Comment:
Given we do still have docker (desktop) around like you mentioned.. should we keep this on the radar but on the outer ring (hold) with a description?

Reply:
Could it be `docker_desktop`/`Docker Desktop` instead of `docker`?

Reply:
Works for me, yeah